packages/engine-rn-web/src/sdks/sdk-webos/index.ts

```diff
@@ -153,9 +163,9 @@ const _configureProject = async (c) => {
 
     addSystemInjects(c, injects);
 
-    const file = path.join(getPlatformProjectDir(c), configFile);
+    const file = path.join(getPlatformProjectDir(c)!, configFile);
 
-    writeCleanFile(file, file, injects, null, c);
+    writeCleanFile(file, file, injects, undefined, c);
```

Comment:
future: we need to reorder the arguments, saw some undefined, undefined before c above.

Reply:
yep. I left it like that intentionally. otherwise I would break any project (ie hook) utilising it. 
we'll use TS method overloading to add new apis